manifests/07-operator.yaml

```diff
@@ -22,6 +22,14 @@ spec:
       - effect: NoSchedule
         key: node-role.kubernetes.io/master
         operator: Exists
+      - key: "node.kubernetes.io/unreachable"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
+      - key: "node.kubernetes.io/not-ready"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120
```

Comment:
should the pods being created by this operator(ie the operands) also have these toleration values?

Reply:
That's upto the operator, if the operand should evict immediately, we can create that in a follow-up PR.